### DISCUSSION TITLE
slowdown for overhangs with arachne?

### DISCUSSION THREAD

**Discussion by John3Di** [OrcaSlicer/OrcaSlicer#141] [Q&A]
I really like the function slowdown for overhangs and would also like to use it with arachne. is this planned for the future or is it even possible?

**Comment by SoftFever** [comment]
As far as I know, Bambu is working on this.
To avoid duplicated effort, I won't work on it.
It is a bit tricky to detect and calculate overhangs for Arachne perimeters. I expect it may take some time for them.
